src/mpid/ch4/netmod/ofi/ofi_am_impl.h

```diff
@@ -6,6 +6,7 @@
 #ifndef OFI_AM_IMPL_H_INCLUDED
 #define OFI_AM_IMPL_H_INCLUDED
 
+#include "mpl_gpu.h"
```

Comment:
Why this header is not included in the `mpl.h`?

Reply:
Err, that was added by a new plugin that I am trying. Should be fixed now.